src/pg/sql/45_observatory_mvt.sql

```diff
@@ -0,0 +1,562 @@
+
+  bounds := cdb_observatory.OBS_GetTileBounds(z, x, y);
+  geom := ST_MakeEnvelope(bounds[1], bounds[2], bounds[3], bounds[4], 4326);
+  ext := ST_MakeBox2D(ST_Transform(ST_SetSRID(ST_Point(bounds[1], bounds[2]), 4326), 3857),
+                      ST_Transform(ST_SetSRID(ST_Point(bounds[3], bounds[4]), 4326), 3857));
+
+  ---------DO---------
+  getmeta_parameters := '[ ';
+  FOREACH measurement IN ARRAY do_measurements LOOP
+    getmeta_parameters := getmeta_parameters || '{"numer_id":"' || measurement || '","geom_id":"' || geography_level || clipped ||'"},';
+  END LOOP;
+  getmeta_parameters := substring(getmeta_parameters from 1 for length(getmeta_parameters) - 1) || ' ]';
+
+  IF use_meta_cache THEN
+    meta := cdb_observatory.OBS_RetrieveMeta(z, geom, getmeta_parameters::json, 1::integer, 1::integer, 1::integer);
+  ELSE
+    meta := cdb_observatory.obs_getmeta(geom, getmeta_parameters::json, 1::integer, 1::integer, 1::integer);
+  END IF;
+
+  IF meta IS NOT NULL THEN
+    SELECT  array_agg(distinct 'observatory.'||numer_tablename) numer_tablenames,
+            string_agg(distinct numer_colname, ',')||',' numer_colnames,
+            string_agg(distinct numer_tablename||'.'||numer_colname, ',')||',' numer_colnames_qualified,
+            string_agg(distinct numer_colname||area_normalization||' '||numer_colname, ',')||',' numer_colnames_normalized,
+            (array_agg(distinct 'observatory.'||geom_tablename))[1] geom_tablenames,
+            (array_agg(distinct geom_colname))[1] geom_colnames,
+            (array_agg(distinct geom_geomref_colname))[1] geom_geomref_colnames,
+            (array_agg(distinct geom_tablename||'.'||geom_geomref_colname))[1] geom_geomref_colnames_qualified,
+            array_agg(distinct numer_tablename||'.'||numer_geomref_colname||'='||geom_tablename||'.'||geom_geomref_colname) geom_relations
+      INTO numer_tablenames_do, numer_colnames_do, numer_colnames_do_qualified, numer_colnames_do_normalized, geom_tablenames, geom_colnames,
+          geom_geomref_colnames, geom_geomref_colnames_qualified, geom_relations_do
+      FROM json_to_recordset(meta)
+        AS x(id TEXT, numer_id TEXT, numer_aggregate TEXT, numer_colname TEXT, numer_geomref_colname TEXT, numer_tablename TEXT,
+            numer_type TEXT, denom_id TEXT, denom_aggregate TEXT, denom_colname TEXT, denom_geomref_colname TEXT, denom_tablename TEXT,
+            denom_type TEXT, denom_reltype TEXT, geom_id TEXT, geom_colname TEXT, geom_geomref_colname TEXT, geom_tablename TEXT,
+            geom_type TEXT, numer_timespan TEXT, geom_timespan TEXT, normalization TEXT, api_method TEXT, api_args JSON);
+
+    IF numer_tablenames_do IS NULL OR numer_colnames_do IS NULL OR numer_colnames_do_qualified IS NULL OR numer_colnames_do_normalized IS NULL
+      OR geom_tablenames IS NULL OR geom_colnames IS NULL OR geom_geomref_colnames IS NULL OR geom_geomref_colnames_qualified IS NULL
+      OR geom_relations_do IS NULL THEN
+      RETURN;
```

Comment:
we should provide some kind of information

Reply:
We can take a look at this behaviour in the next iteration if you agree